collect-connect-google.rst

```diff
@@ -0,0 +1,51 @@
+Connecting to a Google Drive Account
+=======================================
+
+.. admonition:: Before you get started...
+
+  ODK Collect connects to your Google Account using the Google Play Store credentials stored on your Android device. This means that before you can connect Collect to your Google account, you need to `add your account to your device <https://support.google.com/googleplay/answer/2521798?hl=en>`_.
```

Comment:
This isn't required anymore -- you can add an account from the settings.

Reply:
Hmm, might be in the upcoming release, should check on that.

Reply:
I don't have it currently, and I'm pretty sure I'm on the latest available in the app store.